lib/python/vdm/server/HTTPListener.py

```diff
@@ -1630,6 +1668,101 @@ def delete(username, database_id):
         DEPLOYMENT_USERS.remove(current_user[0])
         return jsonify({'status': 1, 'statusstring': "User Deleted"})
 
+class StartDatabaseAPI(MethodView):
+    """Class to handle request to start servers on all nodes of a database."""
+
+    @staticmethod
+    def put(database_id):
+        """
+        Starts VoltDB database servers on all nodes for the specified database
+        Args:
+            database_id (int): The id of the database that should be started
+        Returns:
+            Status string indicating if the database start requesst was sent successfully
+        """
+
+        # TODO: assume deployment file has been sync'ed already????
+        members = []
+        current_database = [database for database in DATABASES if database['id'] == database_id]
+        if not current_database:
+            abort(404)
+        else:
+            members = current_database[0]['members']
+        if not members:
+            return make_response(jsonify({'statusstring': 'No servers configured for the database'}),
+                                             500)
+
+        # Check if there are valid servers configured for all ids
+        for server_id in members:
+            server = [server for server in SERVERS if server['id'] == server_id]
+            if not server:
+                return make_response(jsonify({'statusstring': 'Server details not found for id ' + server_id}),
+                                             500)
+        # Now start each server
+        __host_name__ = socket.gethostname()
+        __host_or_ip__ = socket.gethostbyname(__host_name__)
```

Comment:
thanks for your previous explanation. Are we going to handle servers that have different interfaces with each their own assigned hostname? is the `socket.gethostname()` return the configured value for the http server? What it the hostnane is not associated with an 'A'  record?